test/WebJobs.Script.Tests/Extensions/ScriptJwtBearerExtensionsTests.cs

```diff
@@ -49,6 +55,17 @@ public void CreateTokenValidationParameters_HasExpectedAudience(bool isPlacehold
                 testData[LegionServiceHost] = "1";
             }
 
+            if (isLinuxConsumptionOnAtlas)
```

Comment:
nit: atlas/legion are mutually exclusive, so this should be an if/else

Reply:
Fixed!